dtls.c

```diff
@@ -714,10 +739,37 @@ static const char *dtls_handshake_type_to_name(int type)
   case DTLS_HT_FINISHED:
     return "finished";
   default:
-    return "unknown";
+    pos += print_byte(buf + pos, sizeof(buf) - pos - 2, type & 0xff);
+    assert(pos+2 < sizeof(buf));
+    buf[pos++] = ')';
+    buf[pos] = '\000';
+    return buf;
+  }
+}
+
+static const char *dtls_message_type_to_name(int type)
```

Comment:
whats about returning NULL and use two dtls_debug?

```
tyepAsText=dtls_message_type_to_name(msg[0]);
if (typeAsText) {
   dtls_debug("got '%s' epoch %u sequence %" PRIu64 " (%d bytes)\n",
               typeAsText, dtls_get_epoch(header), pkt_seq_nr, rlen);
} else {
   dtls_debug("got unknown record type (%d) epoch %u sequence %" PRIu64 " (%d bytes)\n",
               msg[0], dtls_get_epoch(header), pkt_seq_nr, rlen);
}
```

Reply:
Gone with the principal of this suggestion and done some other tidy up. Code pushed.